deluge/ui/console/modes/connectionmanager.py

```diff
@@ -174,28 +170,21 @@ def on_resize(self, rows, cols):
     def read_input(self):
         c = self.stdscr.getch()
 
-        if c > 31 and c < 256:
-            if chr(c) == 'q' and self.inlist:
-                return
+        if is_printable_chr(c):
```

Comment:
Yeah I don't think we will be using symbols as key shortcuts ;)

For reference this is the 256 ascii table: http://www.flexcomm.com/library/ASCII256.htm